lib/much-rails/railtie.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module MuchRails
+  class Railtie < Rails::Railtie
+    initializer "much-rails-gem" do |app|
+      require "much-rails/assets"
```

Comment:
@nweathersbee note: I didn't require assets in the main MuchRails require file as I want this to be somewhat optional/late-bound. This says, there is no point requiring the assets stuff if you aren't using the Railtie. FYI.